internal/api/keppel/accounts.go

```diff
@@ -158,18 +158,22 @@ func (a *API) handleDeleteAccount(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	resp, err := a.processor().DeleteAccount(r.Context(), *account, keppel.AuditContext{
+	if account.IsDeleting {
+		respondwith.JSON(w, http.StatusConflict, struct {
+			Error string `json:"error"`
+		}{Error: "account is already set to be deleted"})
+		return
```

Comment:
DELETE is usually an idempotent action in REST APIs, so we don't need to make this an error:
```suggestion
		w.WriteHeader(http.StatusNoContent)
		return
```